hq_superset/views.py

```diff
@@ -115,7 +125,12 @@ def refresh_hq_datasource(domain, datasource_id, display_name):
     schema = get_schema_name_for_domain(domain)
     csv_table = Table(table=datasource_id, schema=schema)
     datasource_defn = get_datasource_defn(provider, token, domain, datasource_id)
-    column_dtypes, date_columns = get_column_dtypes(datasource_defn)
+    column_dtypes, date_columns, array_columns = get_column_dtypes(datasource_defn)
+
+    converters = {column: convert_to_array for column in array_columns}
```

Comment:
Maybe use `column_name` instead of `column`? I.E

`converters = {column_name: convert_to_array for column_name in array_columns}`